README.md

```diff
@@ -5,25 +5,43 @@ An infrastructure template for web applications powered by Terraform.
 **Supported cloud:** AWS. We are looking for contributions to implement support for GCP, Heroku, and Azure!
 
 **Supported flavors:**
-- A `Basic` flavor is in the plan, but not available yet
+- A `Blank` flavor that generates a blank Terraform project.
 - The `Complete` flavor generates the following infrastructure:
 
 ![Diagram of the Complete Infrastructure](/img/diagram_complete.svg?raw=true)
 
 ## Usage
```

Comment:
Should we add a point indicating the need to modify the value for Terraform Cloud org, otherwise, running `terraform plan` on local will fail
<img width="379" alt="image" src="https://github.com/nimblehq/infrastructure-templates/assets/1896814/6ac35e38-6706-47d2-9849-1890e9c6bfe4">

Reply:
probably the next step should be asking the user to provide the provider name in the CLI input :)

Reply:
Yes, we should have the option to enable it 🙏 There is [an issue](https://github.com/nimblehq/infrastructure-templates/issues/191) for it.
Added the note in https://github.com/nimblehq/infrastructure-templates/pull/169/commits/ecf68b4b02cc305a58192c12696a09815cc949a5